docs/cmf_client/ssh-setup.md

```diff
@@ -0,0 +1,25 @@
+# SSH Remote Artifact Repo Setup
+## Steps to set up a SSH Remote Repo
```

Comment:
documentation for configuring cmf with ssh remote should be independent of example get started